vendor.conf

```diff
@@ -51,9 +51,9 @@ github.com/syndtr/gocapability                      d98352740cb2c55f81556b63d4a1
 github.com/urfave/cli                               v1.22.1 # NOTE: urfave/cli must be <= v1.22.1 due to a regression: https://github.com/urfave/cli/issues/1092
 go.etcd.io/bbolt                                    v1.3.5
 go.opencensus.io                                    v0.22.0
-golang.org/x/net                                    f3200d17e092c607f615320ecaad13d87ad9a2b3
+golang.org/x/net                                    ab34263943818b32f575efc978a3d24e80b04bd7
 golang.org/x/sync                                   42b317875d0fa942474b76e1b46a6060d720ae6e
-golang.org/x/sys                                    9dae0f8f577553e0f21298e18926efc9644c281d
+golang.org/x/sys                                    ed371f2e16b4b305ee99df548828de367527b76b
```

Comment:
Note that these are in the "containerd dependencies" section. Wondering if they should stay in that section (and thus need to be updated there first), or if we need some "generic" section, as these are used in various places, not just containerd

Reply:
we'll be merging the cri repo soon into containerd .. agree once something has two major dependencies it's time to move it to a more generic list